components/atoms/ReportProblem.js

```diff
@@ -9,59 +15,169 @@ export default function ReportProblem() {
+            href="mailto:experience@servicecanada.gc.ca"
+          >
+            experience@servicecanada.gc.ca
+          </a>
         </>
       ) : (
-        <>
-          <div className="pt-2">
-            <form action="#" onSubmit={onSubmitHandler}>
-              <div className="container-sm border border-gray-md rounded bg-gray-light-report pb-5">
-                <fieldset>
-                  <legend className="text-center pb-3 pt-4">
-                    <span>
-                      <p className="font-normal text-h3">
-                        Please select all that apply:
-                      </p>
-                    </span>
-                  </legend>
-                  <div className="pl-5">
-                    <div className="absolute">
-                      <label htmlFor="problem1" className="cursor-pointer">
-                        <input
-                          name="problem1"
-                          id="problem1"
-                          type="checkbox"
-                          value="Yes"
-                        />
-                        &nbsp;A link, button or video is not working
-                      </label>
-                      <input name="problem1" type="hidden" value="No" />
-                    </div>
-                  </div>
-                </fieldset>
-                <div className="text-left pl-2 pt-8 text-sm font-display">
-                  <button className="bg-custom-blue-dark pb-1.5 pt-1.5 px-3 ml-2 rounded text-white hover:bg-custom-blue-canadaLight border border-custom-blue-dark focus:ring-1 focus:ring-black focus:ring-offset-1 focus:bg-custom-blue-canadaLight active:bg-custom-blue-reportButtonActive">
-                    Submit
-                  </button>
-                </div>
-              </div>
-            </form>
-          </div>
-        </>
+        <form className="w-full" action="#" onSubmit={onSubmitHandler}>
+          <fieldset>
+            <legend className="text-h3 text-center font-normal mb-6">
+              {t.reportAProblemCheckAllThatApply}
+            </legend>
+            <OptionalTextField
+              checkBoxId="incorrectInformation"
+              textFieldId="incorrectInformationTextField"
+              checkBoxName="incorrectInformation"
+              textFieldName="incorrectInformationText"
+              checkBoxLabel={t.reportAProblemIncorrectInformation}
+              textFieldLabel={t.reportAProblemProvideMoreDetails}
+              uncontrolled={true}
+              multiText={true}
+              textLabelBold={true}
+              wrap="hard"
+              maxLength={750}
+            />
+            <OptionalTextField
+              checkBoxId="unclearInformation"
+              textFieldId="incorrectInformationTextField"
+              checkBoxName="incorrectInformation"
+              textFieldName="incorrectInformationText"
```

Comment:
Pretty sure [in the other Report a problem PR](https://github.com/DTS-STN/Alpha-Site/pull/52), the `name` attributes were all converted to `snake_case` from `camelCase`.

So `incorrectInformationText` becomes `incorrect_information_text`